docs/src/components/header.js

```diff
@@ -1,70 +1,234 @@
+    display: initial;
+    border-right: 1px solid #a9a9a9;
+    height: 8rem;
+    padding: 0.25rem 2.5rem 0.25rem 0;
+    margin-left: -0.5rem;
+    margin-right: 2.5rem;
+  }
+`
+
+const HeaderName = styled.span`
+  font-weight: bold;
+  white-space: nowrap;
+  font-size: 2.5rem;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    font-size: 3rem;
+  }
+`
+
+const Badge = styled.span`
+  padding: 1px 8px;
+  margin-left: 0.5rem;
+  border: solid 2px #fff;
+  border-radius: 2em;
+  font-weight: bold;
+  vertical-align: super;
+  white-space: nowrap;
+  font-size: 1rem;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    font-size: 1.5rem;
+  }
+`
+
+const Nav = styled.nav`
+  padding: 20px;
+  position: fixed;
+  height: 100%;
+  top: 0;
+  width: 100%;
+  background: #313131;
+  z-index: 5;
+  text-align: center;
+
+  &:not(:target) {
+    right: -100%;
+    transition: right 0.5s;
+  }
+
+  &:target {
+    right: 0;
+    transition: right 0.5s;
+  }
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    padding: initial;
+    position: initial;
+    height: initial;
+    top: initial;
+    width: initial;
+    background: initial;
+
+    &:not(:target) {
+      right: initial;
+      transition: initial;
+    }
+
+    &:target {
+      right: initial;
+      transition: initial;
+    }
+  }
+`
+
+const OpenMenuLink = styled.a`
+  color: #fff;
+  align-self: center;
+  text-decoration: none;
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  padding: 2rem 0;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    display: none;
+  }
 `
 
-const Nav = styled.nav``
+const MenuText = styled.span`
+  margin-right: 0.7rem;
+  font-size: 1.7rem;
+`
+
+const CloseLink = styled.a`
+  display: block;
+  color: #fff;
+  text-align: center;
+  margin-bottom: 2.5rem;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    display: none;
+  }
+`
 
-const Navitems = styled.ul`
+const StyledUnsortedList = styled.ul`
   list-style: none;
   margin: 0;
+  padding: 0;
 `
 
-const Navitem = styled.li`
-  display: inline-block;
-  margin: 0 1rem;
+const StyledListItem = styled.li`
+  display: inline;
+  padding-right: 1.5rem;
+  padding-bottom: 1rem;
+  padding-left: 1.5rem;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    padding-bottom: 1.5rem;
+    border-bottom: ${props =>
+      props.active ? '0.5rem solid #fff' : '0.5rem solid #45c2f0'};
+
+    &:last-child {
+      margin-left: auto;
+    }
+  }
+`
+
+const StyledExternalLink = styled(External)`
+  text-decoration: none;
+  color: ${props => (props.active ? '#45c2f0' : '#fff')};
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    color: white;
+  }
 `
 
-export default ({ siteTitle }) => (
+const BoxedListItem = ({ children, href, active, ...props }) => (
+  <Box mb={['0.7rem', '0.1rem']} width={[1, 'auto']} {...props}>
+    <StyledListItem active={active}>
```

Comment:
This is also an `li` not inside a `ul` parent.

Reply:
Changed